src/components/themed/SceneHeader.tsx

```diff
@@ -1,27 +1,30 @@
 import * as React from 'react'
 import { View, ViewStyle } from 'react-native'
 
+import { Space } from '../layout/Space'
 import { cacheStyles, Theme, useTheme } from '../services/ThemeContext'
 import { DividerLine } from './DividerLine'
 import { EdgeText } from './EdgeText'
 
 interface Props {
   title?: string
   children?: React.ReactNode
+  icon?: React.ReactNode
```

Comment:
Wouldn't it make sense to rename the prop when there are other icon props?

Reply:
I don't see what left-right justification has to do with the name of props on a component. Wouldn't it make sense to rename the props when there are other icon props because only then does it need more distinction?

Reply:
condense into one children prop again, make sure no existing users of this component will have its styling affected if we update the children styling to space-inbetween

Reply:
Rename to `tertiary`.